File: Fibonacci.java
class Fibonacci{

  public static void main(String [] args){
    int x;
    System.out.println(getFib(x));
  }

  /**
  determines the Fibonacci sequence
  */
  public int getFib(int n){
    if (n <= 1){
      return n;
    }else{
      return getFib(n-1) + getFib(n-2);
    }
  }
}

File: fibonacci.js
var fs  = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line !== "") { // ignore empty lines
        console.log(fibonacci(line));
    }
});

function fibonacci(number){
  if (number <= 1){
    return number;
  }else{
    return getFib(number - 1) + getFib(number - 2);
  }
}
